src/args.rs

```diff
@@ -17,40 +17,43 @@ use strum::{EnumString, IntoStaticStr, VariantArray};
 #[derive(EnumString, IntoStaticStr, VariantArray, Debug, Clone, Copy, PartialEq, Eq)]
 #[strum(serialize_all = "kebab-case")]
 pub enum Arg {
+    AutoOrient,
```

Comment:
I sorted all those alphabetically. Can revert obvsly ... :)

Reply:
The sorting is kind of a mess but at least it's the same in both the original list and the matches. I'd like to get everything sorted alphabetically (both this enum and the matches) but it's a bit of work so I procrastinated that and opened a feature request for rust-analyzer instead: https://github.com/rust-lang/rust-analyzer/issues/20766

Reply:
Haha no worries, done